js/languages/en-US.json

```diff
@@ -741,5 +741,991 @@
     "10": "(GMT +10:00) Eastern Australia, Guam, Vladivostok",
     "11": "(GMT +11:00) Magadan, Solomon Islands, New Caledonia",
     "12": "(GMT +12:00) Auckland, Wellington, Fiji, Kamchatka"
+  },
+  "countries": {
```

Comment:
@jjeffryes Using our translation software, is it possible to break up individual langs into multiple files? If so, I think it would be worthwhile to break out the `countries` data, since it's pretty darn lengthy.